.config/zepter.yaml

```diff
@@ -25,9 +25,13 @@ workflows:
         '--show-path',
         '--quiet',
       ]
-  # Same as `check`, but with the `--fix` flag.
+  # The umbrella crate uses more features, so we to check those too:
```

Comment:
Just a tiny thing: I guess the word `need` or `have` is missing: `so we need/have to check`